zetaclient/evm/evm_client.go

```diff
@@ -1161,13 +1127,6 @@ func (ob *ChainClient) observeERC20Deposited(startBlock, toBlock uint64) uint64
 // returns the last block successfully scanned
 func (ob *ChainClient) observeTssRecvd(startBlock, toBlock uint64, flags observertypes.CrosschainFlags) uint64 {
```

Comment:
Is there a reason why we removed the validation of received txns to TSS, for example donations? Also the vote to the core is removed.

Reply:
Thanks, I noticed after.